include/psp2kern/net/net.h

```diff
@@ -1,4 +1,9 @@
 /**
+ * \kernelgroup{SceNet}
+ * \usage{psp2kern/net/net.h,-lSceNetPs_stub}
```

Comment:
SceNetPs is right.
https://github.com/vitasdk/vita-headers/blob/11abeb6/db.yml#L3111